### PR TITLE
Add default pr & issue templates

### PR DESCRIPTION
**Pull request by z4kn4fein**

### CODE REVIEW

File: ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+If you are reporting a bug related to one of our SDK's, please provide the following information.
+
+### SDK version
+
+The version of this SDK that you are using.
+
+### SDK configuration
+
+If it differs from the default, add your SDK configuration here e.g. **polling mode**, **refresh interval** or **custom cache**.
```
Comment (sigewuzhere) [inline ISSUE_TEMPLATE/bug_report.md:38]: suggestion: Please copy paste the client initialization code snippet, so we can see how you are using the SDK.
<example code snippet might help here>

File: ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+### Logs
+
+If applicable, add the related SDK's log output here.
+
+### Language version
```
Comment (sigewuzhere) [inline ISSUE_TEMPLATE/bug_report.md:44]: Suggestion: Programming language version

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+### Related issues
```
Comment (sigewuzhere) [inline pull_request_template.md:1]: Is this the template for our internal PRs as well?
If yes: see trello ticket description
If no: nevermind me

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

File: ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (z4kn4fein) [reply ISSUE_TEMPLATE/bug_report.md]: Done

File: ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (z4kn4fein) [reply ISSUE_TEMPLATE/bug_report.md]: I think it's pretty obvious that we want a programming language version here. But I extended with the term `Framework` cause in some cases that's more accurate than the language.

File: pull_request_template.md (hunk 3 above)
Comment (z4kn4fein) [reply pull_request_template.md:1]: Nope, for internal PRs I'll create separate templates in every repo

**Review by sigewuzhere — APPROVED** [review]

**Pull request merged**